tools/benchmark/src/mocha/customOutputRunner.ts

```diff
@@ -0,0 +1,89 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Test } from "mocha";
+
+import type {
+	BenchmarkDescription,
+	HookArguments,
+	MochaExclusiveOptions,
+	Titled,
+} from "../Configuration";
+import type { BenchmarkData } from "../ResultTypes";
+import { timer } from "../timer";
+
+/**
+ * Options to configure a benchmark that reports custom measurements.
+ *
+ * @public
+ */
+export interface CustomBenchmarkOptions
+	extends Titled,
+		BenchmarkDescription,
+		HookArguments,
+		MochaExclusiveOptions {
+	/**
+	 * Runs the benchmark.
+	 */
+	run: (reporter: IMeasurementReporter) => void | Promise<unknown>;
+}
+/**
```

Comment:
I generally like a blank line between top level items in a file.
```suggestion

/**
```

Reply:
Me too. Update it